scripts/gen_openapi.sh

```diff
@@ -29,89 +29,55 @@ export VAULT_ADDR=http://127.0.0.1:8200
-        break
-    elif [ $codeLinesStarted = true ] && [[ $line == *"consts.Deprecated"* || $line == *"consts.PendingRemoval"* || $line == *"consts.Removed"* ]] ; then
-        auth_plugin_previous=""
-    elif [ $codeLinesStarted = true ] && [[ $line =~ ^\s*\"(.*)\"\:.*$ ]] ; then
-        auth_plugin_current=${BASH_REMATCH[1]}
-
-        if [[ -n "${auth_plugin_previous}" ]] ; then
-            echo "enabling auth plugin: ${auth_plugin_previous}"
-            vault auth enable "${auth_plugin_previous}"
-        fi
-
-        auth_plugin_previous="${auth_plugin_current}"
-    fi
-done <../../vault/helper/builtinplugins/registry.go
-
-if [[ -n "${auth_plugin_previous}" ]] ; then
-    echo "enabling auth plugin: ${auth_plugin_previous}"
-    vault auth enable "${auth_plugin_previous}"
-fi
+vault auth enable "alicloud"
+vault auth enable "approle"
+vault auth enable "aws"
+vault auth enable "azure"
+vault auth enable "centrify"
+vault auth enable "cert"
+vault auth enable "cf"
+vault auth enable "gcp"
+vault auth enable "github"
+vault auth enable "jwt"
+vault auth enable "kerberos"
+vault auth enable "kubernetes"
+vault auth enable "ldap"
+vault auth enable "oci"
+vault auth enable "oidc"
+vault auth enable "okta"
+vault auth enable "pcf"
+vault auth enable "radius"
+vault auth enable "userpass"
 
 # Enable secrets plugins
-codeLinesStarted=false
-
-while read -r line; do
-    if [[ $line == *"logicalBackends:"* ]] ; then
-        codeLinesStarted=true
-    elif [[ $line == *"addExternalPlugins("* ]] ; then
-        break
-    elif [ $codeLinesStarted = true ] && [[ $line == *"consts.Deprecated"* || $line == *"consts.PendingRemoval"* || $line == *"consts.Removed"* ]] ; then
-        secrets_plugin_previous=""
-    elif [ $codeLinesStarted = true ] && [[ $line =~ ^\s*\"(.*)\"\:.*$ ]] ; then
-        secrets_plugin_current=${BASH_REMATCH[1]}
-
-        if [[ -n "${secrets_plugin_previous}" ]] ; then
-            echo "enabling secrets plugin: ${secrets_plugin_previous}"
-            vault secrets enable "${secrets_plugin_previous}"
-        fi
-
-        secrets_plugin_previous="${secrets_plugin_current}"
-    fi
-done <../../vault/helper/builtinplugins/registry.go
-
-if [[ -n "${secrets_plugin_previous}" ]] ; then
-    echo "enabling secrets plugin: ${secrets_plugin_previous}"
-    vault secrets enable "${secrets_plugin_previous}"
-fi
+vault secrets enable "ad"
+vault secrets enable "alicloud"
+vault secrets enable "aws"
+vault secrets enable "azure"
+vault secrets enable "consul"
+vault secrets enable "database"
+vault secrets enable "gcp"
+vault secrets enable "gcpkms"
+vault secrets enable "kubernetes"
+vault secrets enable "kv"
+vault secrets enable "ldap"
+vault secrets enable "mongodbatlas"
+vault secrets enable "nomad"
+vault secrets enable "openldap"
+vault secrets enable "pki"
+vault secrets enable "rabbitmq"
+vault secrets enable "ssh"
+vault secrets enable "terraform"
+vault secrets enable "totp"
+vault secrets enable "transit"
```

Comment:
[non-blocking]
same here
```suggestion
readonly SECRET_PLUGINS=(
  # SECRET_PLUGINS - START
  ad
  alicloud
  aws
  azure
  consul
  database
  gcp
  gcpkms
  kubernetes
  kv
  ldap
  mongodbatlas
  nomad
  openldap
  pki
  rabbitmq
  ssh
  terraform
  totp
  transit
  # SECRET_PLUGINS - END
)

for secret_plugin in "${SECRET_PLUGINS[@]}"; do
  vault secrets enable "${secret_plugin}"
done
```